CMakeLists.txt

```
@@ -39,6 +40,12 @@ add_custom_target(luatest
   COMMENT "Run regression tests"
 )
 
+add_custom_target(luatest-no-coverage
+  COMMAND ${LUATEST} -v
```

`shuffle` is not related to the coverage.
```suggestion
  COMMAND ${LUATEST} -v --shuffle all:${seed}
```